src/components/ContactForm.tsx

```
@@ -31,15 +32,15 @@ function SubmitButton() {
 // Success Message Component
 function SuccessMessage({ onReset, className = '' }: { onReset: () => void; className?: string }) {
   return (
-    <div className={`glass-card border-green-700/50 text-center ${className}`}>
+    <div className={cn("glass-card p-6 rounded-xl border border-border border-green-700/50 text-center", className)}>
       <div className="mb-6">
         <svg className="w-16 h-16 mx-auto text-green-400" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)